### PR TITLE
[IOTDB-399]修改了一些文档

### PR DESCRIPTION
**Pull request by SailVR**
# https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-399?filter=allopenissues

### CODE REVIEW

File: docs/Documentation-CHN/UserGuide/0-Get Started/3-Publication.md [hunk 1]
```diff
@@ -29,3 +29,9 @@ Apache IoTDB 始于清华大学软件学院。IoTDB是一个用于管理大量
 * [Matching Consecutive Subpatterns over Streaming Time Series](https://link.springer.com/chapter/10.1007/978-3-319-96893-3_8), Rong Kang and Chen Wang and Peng Wang and Yuting Ding and Jianmin Wang. APWeb/WAIM 2018.
 * [KV-match: A Subsequence Matching Approach Supporting Normalization and Time Warping](https://www.semanticscholar.org/paper/KV-match%3A-A-Subsequence-Matching-Approach-and-Time-Wu-Wang/9ed84cb15b7e5052028fc5b4d667248713ac8592), Jiaye Wu and Peng Wang and Chen Wang and Wei Wang and Jianmin Wang. ICDE 2019.
 * 我们还研发了面向时间序列数据库的Benchmark工具： [https://github.com/thulab/iotdb-benchmark](https://github.com/thulab/iotdb-benchmark)
+
+# Benchmark工具
+
+我们还研发了面向时间序列数据库的Benchmark工具： 
+
+https://github.com/thulab/iotdb-benchmark
```
Comment (samperson1997) [inline docs/Documentation-CHN/UserGuide/0-Get Started/3-Publication.md]: Content is duplicated here. 
这里的内容重复了

File: docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md [hunk 2]
```diff
@@ -21,4 +21,46 @@
...
+下载命令为：
+
+```
+pip install apache-iotdb
+```
+
+#### 方案2: 使用我们提供的编译脚本
+
+如果你在路径中添加了Thrift可执行文件，则可以运行`client-py / compile.sh`或
+  `client-py \ compile.bat`，或者你必须对其进行修改以将变量`THRIFT_EXE`设置为指向你的可执行文件。 这将在`target`文件夹下生成节俭源，你可以将其添加到你的`PYTHONPATH`，以便你可以在代码中使用该库。 请注意，脚本通过相对路径找到节俭的源文件，因此，如果将脚本移动到其他位置，它们将不再有效。
```
Comment (samperson1997) [inline docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md]: “节俭的源文件”
Typo

File: docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md [hunk 3]
```diff
@@ -309,8 +309,384 @@ select last id, status, temperature from root.ln.wf01 disable align
 |  9   | root.ln.wf01.wt01.temperature| 35.7  |
 ```
 
+### 自动填充
+
+在IoTDB的实际使用中，当进行时间序列的查询操作时，可能会出现在某些时间点值为null的情况，这会妨碍用户进行进一步的分析。 为了更好地反映数据更改的程度，用户希望可以自动填充缺失值。 因此，IoTDB系统引入了自动填充功能。
+
+自动填充功能是指对单列或多列执行时间序列查询时，根据用户指定的方法和有效时间范围填充空值。 如果查询点的值不为null，则填充功能将不起作用。
+
+> 注意：在当前版本中，IoTDB为用户提供两种方法：Previous 和Linear。 previous 方法用前一个值填充空白。 Linear方法通过线性拟合来填充空白。 并且填充功能只能在执行时间点查询时使用。
```
Comment (samperson1997) [inline docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md]:
```suggestion
> 注意：在当前版本中，IoTDB为用户提供两种方法：Previous 和Linear。 Previous 方法用前一个值填充空白。 Linear方法通过线性拟合来填充空白。 并且填充功能只能在执行时间点查询时使用。
```

File: docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md [hunk 4]
```diff
@@ -309,8 +309,384 @@ select last id, status, temperature from root.ln.wf01 disable align
...
+
+当SLIMIT子句的参数SOFFSET不小于可用时间序列数时，系统将提示错误。 例如，执行以下SQL语句：
+
+```
+select * from root.ln.wf01.wt01 where time > 2017-11-01T00:05:00.000 and time < 2017-11-01T00:12:00.000 slimit 1 soffset 2
+
+```
 
-## 数据维护
+SQL语句将不会执行，并且相应的错误提示如下：
```
Comment (samperson1997) [inline docs/Documentation-CHN/UserGuide/5-Operation Manual/2-DML (Data Manipulation Language).md]: Here a figure is missed. Link: https://user-images.githubusercontent.com/13203019/51578237-cd33f300-1ef7-11e9-9aef-2a717c56ab54.jpg
这里遗漏了一张图片

File: docs/Documentation-CHN/UserGuide/0-Get Started/1-QuickStart.md [hunk 5]
```diff
@@ -281,3 +281,13 @@ Windows系统停止命令如下：
 ```
 > $sbin\stop-server.bat
 ```
+
+## 只建立客户端
+
+在incubator-iotdb的跟路径下：
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/0-Get Started/1-QuickStart.md]:
```suggestion
在incubator-iotdb的根路径下：
```

File: docs/Documentation-CHN/UserGuide/0-Get Started/1-QuickStart.md [hunk 6]
```diff
@@ -281,3 +281,13 @@ Windows系统停止命令如下：
...
+
+```
+> mvn clean package -pl client -am -DskipTests
+```
+
+构建后，IoTDB客户端将位于文件夹“ client / target / iotdb-client- {project.version}”下。
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/0-Get Started/1-QuickStart.md]:
```suggestion
构建后，IoTDB客户端将位于文件夹 “client/target/iotdb-client-{project.version}”下。
```

File: docs/Documentation-CHN/UserGuide/4-Client/2-Programming - Native API.md [hunk 7]
```diff
@@ -23,6 +23,8 @@
 
 # 编程 - 原生接口
 
+## 使用
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/4-Client/2-Programming - Native API.md]: no need, remove this title in English version

File: docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md [hunk 8]
```diff
@@ -21,4 +21,46 @@
...
+
+下载命令为：
+
+```
+pip install apache-iotdb
+```
+
+#### 方案2: 使用我们提供的编译脚本
+
+如果你在路径中添加了Thrift可执行文件，则可以运行`client-py / compile.sh`或
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md]:
```suggestion
如果你在路径中添加了Thrift可执行文件，则可以运行`client-py/compile.sh`或
```

File: docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md [hunk 9]
```diff
@@ -21,4 +21,46 @@
...
+  `client-py \ compile.bat`，或者你必须对其进行修改以将变量`THRIFT_EXE`设置为指向你的可执行文件。 这将在`target`文件夹下生成节俭源，你可以将其添加到你的`PYTHONPATH`，以便你可以在代码中使用该库。 请注意，脚本通过相对路径找到节俭的源文件，因此，如果将脚本移动到其他位置，它们将不再有效。
+
+#### 方案3：thrift的基本用法
+
+或者，如果您了解thrift的基本用法，则只能在以下位置下载Thrift源文件：
+`service-rpc \ src \ main \ thrift \ rpc.thrift`，并且只需使用`thrift -gen py -out ./target/iotdb rpc.thrift`生成python库。
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md]:
```suggestion
`service-rpc\src\main\thrift\rpc.thrift`，并且只需使用`thrift -gen py -out ./target/iotdb rpc.thrift`生成python库。
```

File: docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md [hunk 10]
```diff
@@ -21,4 +21,46 @@
...
+
+### 4. Use Example
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md]:
```suggestion
### 4. 示例代码
```

File: docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md [hunk 11]
```diff
@@ -21,4 +21,46 @@
...
+
+我们在`client-py / src/ client_example.py`中提供了一个示例，说明如何使用Thrift库连接到IoTDB，请先仔细阅读，然后再编写自己的代码。
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/4-Client/4-Programming - Other Languages.md]:
```suggestion
我们在`client-py/src/client_example.py`中提供了一个示例，说明如何使用Thrift库连接到IoTDB，请先仔细阅读，然后再编写自己的代码。
```

File: docs/Documentation/UserGuide/4-Client/3-Programming - JDBC.md [hunk 12]
```diff
@@ -29,6 +29,14 @@
 * JDK >= 1.8
 * Maven >= 3.1
 
+## Package only JDBC projects
+
+Execute the following command in the root directory:
+
+```
+mvn clean package -pl jdbc -am -Dmaven.test.skip=true
```
Comment (qiaojialin) [inline docs/Documentation/UserGuide/4-Client/3-Programming - JDBC.md]:
```suggestion
mvn clean package -pl jdbc -am -DskipTests
```

change the English version also

**Pull request merged**